docs/tutorial.rst

```diff
@@ -224,8 +224,6 @@ The following, for example, only downloads the first file of the request:
 .. code-block:: python
 
     >>> export_request.download(out_dir, index=0)  # doctest: +REMOTE_DATA
-                                                  record                                                url                                           download
-    0  hmi.V_45s[2016.04.01_00:00:00_TAI][2]{Dopplerg...  http://jsoc.stanford.edu/SUM58/D803708321/S000...  ...
```

Comment:
Did this output nothing?

Reply:
Well doctests don't capture log output apparently

Reply:
lmao